src/lib/snark_work_lib/snark_work_lib.ml

```diff
@@ -0,0 +1,6 @@
+module Work = struct
```

Comment:
Just a note: this is temporary, we'll have a good layout after every module is tracked on it's own